.swiftformat

```diff
@@ -7,9 +7,13 @@
 --disable hoistAwait
 --disable hoistTry
 --disable redundantReturn
+--disable hoistPatternLet
```

Comment:
this is for `value` in:
case string(value: String)
case array(value: [String])
for nicer json decosing